packages/gatsby-telemetry/src/repository-id.ts

```diff
@@ -83,8 +87,45 @@ const getRepositoryFromNetlifyEnv = (): IRepositoryId | null => {
   return null
 }
 
+function getRepositoryFromHerokuEnv(): IRepositoryId | null {
+  if (getCIName() !== `Heroku`) {
+    return null
+  }
+
+  const proc = readdirSync(`/proc`).filter(dir => Number.isFinite(Number(dir)))
```

Comment:
Maybe add a comment here of what you are doing.

Reply:
done